starter/schemaorg/books/eval/annotated.tsv

```diff
@@ -51,7 +51,7 @@
 64	has this book won any notable awards ?	[ award ] of @org.schema.Book . Book ( ) ;
 65	how many books by poe are rated at least 4 stars ?	count ( @org.schema.Book . Book ( ) filter aggregateRating . ratingValue >= 4 && author == null ^^org.schema.Book:Person ( " poe " ) ) ;
 67	show me all books with at least NUMBER_0 pages and at least a 4 star review	@org.schema.Book . Book ( ) filter aggregateRating . ratingValue >= 4 && numberOfPages >= NUMBER_0 ;
-68	which book has won the most awards ?	sort ( count ( award ) desc of @org.schema.Book . Book ( ) ) [ 1 ] ;
+68	which book has won the most awards ?	[ * , count ( award ) ] of sort ( count ( award ) desc of @org.schema.Book . Book ( ) ) [ 1 ] ;
```

Comment:
Are these changes correct? I don't think we generate this code in synthesis, and the code with just the sort looks more correct to me.